WX-1627 Belated addition of v87 changelog entries (#7436)
broadinstitute/cromwell: CHANGELOG.md

## 87 Release Notes

### GCP Batch

>  * Added NVidia driver install ([#7235](https://github.com/broadinstitute/cromwell/pull/7235}))

Reviewer: ```suggestion
 * Added Nvidia driver install (default 418) ([#7235](https://github.com/broadinstitute/cromwell/pull/7235}))
```
The driver version matters for what hardware it supports.

[3] Cromwell itself uses crc32c hashes for call caching and is not affected

### Other Improvements

>  * In certain cases DRS downloads have been found to hang forever Cromwell will now time these out ([#7416](https://github.com/broadinstitute/cromwell/pull/7416))

Reviewer: ```suggestion
 * In certain cases DRS downloads have been found to hang forever. Cromwell will now time these out. ([#7416](https://github.com/broadinstitute/cromwell/pull/7416))
```

 * Increased default Akka `client.parsing.max-response-reason-length` to 1024 ([#7406](https://github.com/broadinstitute/cromwell/pull/7406))
 * Workflow Completion Callback bodies now include fully-qualified output names ([#7234](https://github.com/broadinstitute/cromwell/pull/7234))
 * Improved workflow abort error handling ([#7245](https://github.com/broadinstitute/cromwell/pull/7245))
 * Improved logging for troubleshooting ([#7246](https://github.com/broadinstitute/cromwell/pull/7246)) ([#7253](https://github.com/broadinstitute/cromwell/pull/7253)) ([#7388](https://github.com/broadinstitute/cromwell/pull/7388))

>  * Support for Intel Ice Lake chips in PAPI backend ([#7252](https://github.com/broadinstitute/cromwell/pull/7252))

Reviewer: ```suggestion
 * Support for Intel Ice Lake chips in Life Sciences backend ([#7252](https://github.com/broadinstitute/cromwell/pull/7252))
```
Lingo leakage – people who started with Cromwell in the last few years don't know we call all versions of  the API "PAPI"
